docs/quickstart/clustering.md

```diff
@@ -0,0 +1,129 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Clustering
+
+In this guide, you'll learn how to use `sklearn-evaluation`, and `sklearn` to evaluate clustering models.
+
++++
+
+## Sample clustering model
+
+Let's generate some sample data with 5 clusters; note that in most real-world use cases, you won't have ground truth data labels (which cluster a given observation belongs to). However, in this case, the ground truth data is available, which will help us explain the concepts more clearly.
+
+```{code-cell} ipython3
+from sklearn.datasets import make_blobs
+
+X, y = make_blobs(n_samples=1000,
```

Comment:
I'd use an actual data set so it's interesting vs some generated data.

Reply:
i tried several actual datasets but the clusters were not well-separated and it was hard to explain the concepts.

Reply:
Alright, whichever you prefer.